### PR TITLE
Update husky to 5.0.4

### PR DESCRIPTION
**Pull request by aeifn**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -125,6 +114,6 @@
         "xmldom": "0.2.1"
     },
     "engines": {
-        "node": "15.2.0"
+        "node": "15.x"
```
Comment (klimashkin) [inline package.json:117]: I think keeping the exact version is still a good idea. Is there a reason to change?

File: package.json [hunk 2]
```diff
@@ -35,17 +35,6 @@
...
-        "hooks": {
-            "pre-commit": "lint-staged"
-        }
-    },
-    "lint-staged": {
-        "*.js": [
-            "eslint --fix",
-            "git add"
-        ]
-    },
```
Comment (klimashkin) [inline package.json:48]: Where is `lint-staged` going?

File: package.json (hunk 1 above)
Comment (aeifn) [reply package.json:117]: Actual `node` version is set in https://github.com/PastVu/node/blob/master/Dockerfile, that image changes rarely.
But sometimes it is easier to run `npm`, `npx` and other utils on locally installed version of node, which may differ by minor. So, I left only major part.
Also, local version of npm is used here https://github.com/PastVu/pastvu/blob/master/.husky/pre-commit.
Maybe, ideologically better always use docker-compose to run such tools, but its initialisation is slower.

File: package.json (hunk 2 above)
Comment (aeifn) [reply package.json:48]: According to [husky 5 installation guide](https://typicode.github.io/husky/#/?id=add-a-hook) it goes to https://github.com/PastVu/pastvu/blob/master/.husky/pre-commit.

File: package.json (hunk 1 above)
Comment (klimashkin) [reply package.json:117]: Let's them keep the version exact and in sync, we don't want developers, ci and production to use different minor versions of node and see different results.
To make sure that local version is correct you can use [nvs](https://github.com/jasongin/nvs) with `auto` flag and `.node-version` or `.nvmrc` files

Comment (aeifn) [reply package.json:117]: Ok, I will make 'Sync node version within entire project' pull request soon.

**Pull request merged**